packages/@o3r/styling/package.json

```diff
@@ -133,7 +133,7 @@
     "sass": "~1.69.5",
     "semver": "^7.5.2",
     "stylelint": "^15.10.1",
```

Comment:
missing this one

Reply:
yeah, this one breaks the unit-tests on stylelint-plugin